src/utils/tailwind-config-color.js

```diff
@@ -1,13 +1,14 @@
-import resolveConfig from 'tailwindcss/resolveConfig'
 import tailwindConfig from '../../tailwind.config'
 
 // Access theme color values for dynamically applying inline styles in React
 export const getTailwindConfigColor = (color) => {
-  const fullConfig = resolveConfig(tailwindConfig)
+  const allColors = {
+    ...tailwindConfig.theme.colors,
+    ...tailwindConfig.theme.extend.colors,
+  }
```

Comment:
this achieves the exact same result as `resolveConfig(tailwindConfig).theme.colors`